src/server/mod.rs

```diff
@@ -16,6 +13,41 @@ use handlers;
 use handlers::ServerState;
 use settings::Settings;
 
+macro_rules! init_routes {
+    ($app:expr) => {
+        $app.resource("{uid}/info/collections", |r| {
+            r.method(http::Method::GET).with(handlers::collections);
+        }).resource("{uid}/info/collection_counts", |r| {
+                r.method(http::Method::GET)
```

Comment:
The indentation here looks all kinds of wrong, I know, but `rustfmt` insists this is how it should be. 😕